etc/kube/start-kube-docker.sh

```diff
@@ -13,14 +13,6 @@ docker run \
     --net=host \
     --pid=host \
     --privileged=true \
-    gcr.io/google_containers/hyperkube:v1.5.1 \
-    /hyperkube kubelet \
-        --containerized \
-        --hostname-override="127.0.0.1" \
-        --address="0.0.0.0" \
-        --api-servers=http://localhost:8080 \
-        --cluster_dns=10.0.0.10 \
-        --cluster_domain=cluster.local \
-        --config=/etc/kubernetes/manifests \
-        --allow-privileged=true
+    gcr.io/google_containers/hyperkube:v1.6.2 \
+    sh "/rootfs/$PWD/etc/kube/internal.sh"
```

Comment:
I considered doing it that way, this seemed slightly easier just because there wouldn't need to be an extra docker image that got built on all of our machines and could potentially get cached which would be kinda weird. I think they're pretty equivalent complexity wise though.